tests/Tests/ORM/Functional/QueryParameterTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Tests\Models\CMS\CmsUser;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use PHPUnit\Framework\Attributes\Group;
+
+#[Group('GH-11278')]
+final class QueryParameterTest extends OrmFunctionalTestCase
+{
+    private int $userId;
+
+    protected function setUp(): void
+    {
+        $this->useModelSet('cms');
+
+        parent::setUp();
+
+        $this->_em->getConnection()
+            ->executeStatement('DELETE FROM cms_users WHERE username IN (\'john\', \'jane\', \'bill\')');
```

Comment:
Is there a point to this? Doesn't `OrmFunctionalTestCase::tearDown()` take care of that?

Reply:
It should, yeah. But I got unique key violations locally. But I cannot reproduce them anymore. Let's try again without that statement.

Reply:
Looks like it worked. 🙂